config: typo and rmv Marco Acorte’s list because redundant

GiddyGoatGaming opened the pull request:
Marco Acorte’s list is already included in Hagezi’s threat intelligence so keeping it alongside is redundant

ignoramous commented on config.json:
```diff
@@ -1062,7 +1061,7 @@
       "group": "Privacy",
       "subg": "",
-      "format": "domains",
+      "format": "hosts",
```
I see domains in the `url` not hosts...?

GiddyGoatGaming replied:
> I see domains in the `url` not hosts...?

This is the thing you said to leave at hosts here https://github.com/serverless-dns/blocklists/pull/121#discussion_r1064004539 .

ignoramous replied:
oh its a mix? That's fine... I am dog chasing my own tail, wouldn't you know ;)

GiddyGoatGaming replied:
> oh this list is a mix? That's fine... I am a dog chasing my own tail, wouldn't you know ;)

🤣

ignoramous commented on config.json:
```diff
@@ -637,13 +637,12 @@
+      "format": ["wildcard", "hosts"],
       "url": [
         "https://raw.githubusercontent.com/hagezi/dns-blocklists/main/wildcard/tif.txt",
-        "https://raw.githubusercontent.com/marco-acorte/antispam-it/main/antispam-it.txt",
```
We can leave it up in case Hagezi's lists alllowlist / whitelist a domain?

ignoramous commented on config.json:
```diff
@@ -1594,10 +1593,10 @@
       "url": [
         "https://raw.githubusercontent.com/nextdns/metadata/master/privacy/native/xiaomi",
-        "https://github.com/unknownFalleN/xiaomi-dns-blocklist/blob/master/xiaomi_dns_block.lst"],
+        "https://raw.githubusercontent.com/hagezi/dns-blocklists/main/wildcard/native.huawei.txt"],
```
Why Huawei in Xiaomi entry? Mistyped?

GiddyGoatGaming commented:
Sorry about all that, I got confused between huawei and xiaomi (i always have, Both are Chinese phone companies🤣). I moved the hagezi huawei list into the huawei list and moved back the xiaomi list. And I added back Marco’s list👍

ignoramous commented on config.json:
```diff
@@ -637,14 +637,14 @@
       "group": "Security",
       "subg": "Hagezi",
-      "format": ["wildcard", "domains", "hosts"],
+      "format": ["wildcard", "hosts"],
```
missing corresponding entry here. possibly, `"domains"`?

GiddyGoatGaming replied:
> missing corresponding entry here. possibly, `"domains"`?

Nice eye👀. Also, I haven’t made this many mistakes adding/switching lists since my first commit for this repo. I’m so disappointed in myself😂

ignoramous replied:
That's okay. Don't fret about it. There aren't tests here and this work is rote and manual. Happens more often than you think and never gets any more error-free with time or anything.

GiddyGoatGaming commented:
Thanks!
